db/test/versions/huge_bufs.js

```diff
@@ -0,0 +1,2207 @@
+// Data from a real DB.  See 0008_test.js
```

Comment:
nit: I would probably put this inside `versions/fixtures` to keep things clean in that directory.